src/main/avro/lessonStatusItem.avsc

```diff
@@ -0,0 +1,18 @@
+{
+"namespace": "event.lesson_status",
+"type": "record",
+"name": "LessonStatusItem",
+"fields": [
+   {"name": "userUuid", "type": "string"},
+   {"name": "courseUuid", "type": "string"},
+   {"name": "unitId", "type": "int"},
+   {"name": "lessonUuid", "type": "string"},
+   {"name": "completionStatus", "type": {"type": "enum", "name": "CompletionStatus", "symbols": ["NOT_STARTED", "IN_PROGRESS", "COMPLETED"]}},
+   {"name": "score", "type": "int"},
+   {"name": "time", "type": "int"},
```

Comment:
**MINOR**. I know it's an experiment and we don't use real data, but maybe `long` is better?